workshop.wwwapi/Data/DatabaseContext.cs

```diff
@@ -17,10 +17,27 @@ public DatabaseContext(DbContextOptions<DatabaseContext> options) : base(options
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             //TODO: Appointment Key etc.. Add Here
-            
 
-            //TODO: Seed Data Here
+            modelBuilder.Entity<Appointment>().HasKey(e => new { e.PatientId, e.DoctorId });
```

Comment:
your key might need to include the datetime so a patient can be booked on the same date + different time with the same doctor (rare, but possiblE?)